### PR TITLE
Adding backlinks to maths page

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/rulIKfb4/2836-indexing-the-maths-subject-specific-page

### CODE REVIEW

File: app/views/content/blog/abigails-career-progression-story.md [hunk 1]
```diff
@@ -66,4 +66,4 @@ My advice for anyone thinking of getting into teaching is to remember that you w
 
     Induction used to last 1 year, and teachers would be called 'newly qualified teachers' (NQTs).
 
-If you’ve been inspired by Abigail’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression).
+If you’ve been inspired by Abigail’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression). If you have a passion for maths, too, find out more about [becoming a maths teacher](/subjects/maths).
```
Comment (Tomcowan1) [inline app/views/content/blog/abigails-career-progression-story.md:69]: Would change the last sentence to "Or, find out more about [becoming a maths teacher]

File: app/views/content/blog/from-teaching-internship-to-assistant-head-of-maths.md [hunk 2]
```diff
@@ -62,4 +62,4 @@ One of the most rewarding moments of this year was when a student chose to come
 
 One of the most gratifying things about being a teacher is when a student, who at 11 years old was asking you what would happen if there was no gravity, leaves school at 18 to do a physics, maths or engineering degree. You know you’ve shaped their life by making them passionate about your subject. I’ve also had cards at the end of one school year from students not going on to study anything to do with physics, but saying they would continue to read about and engage with it for the rest of their lives.
 
-If you’ve been inspired by Ben’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression).
+If you’ve been inspired by Ben’s story, find out more about [career progression opportunities and teacher pay scales](/salaries-and-benefits#career-progression). If you have a passion for maths, too, find out more about [becoming a maths teacher](/subjects/maths).
```
Comment (Tomcowan1) [inline app/views/content/blog/from-teaching-internship-to-assistant-head-of-maths.md:65]: Same as above

**Review by Tomcowan1 — COMMENTED** [review]
Thanks @EllieNodder have left a couple of comments. Nice work updating the funding widget! A developer in the making :-)

**Comment by gemmadallmandfe** [comment]
Looks great to me @EllieNodder!

@Tomcowan1 Are you happy for this to be merged now? If so, I will approve and merge first thing tomorrow

My only comment would be on the funding widget - whether we need to space things out a bit for clarity in the **Next steps** section but let's deal with that one separately (I'll raise a separate ticket) and get these backlinks into production so we can start getting some traffic to this page.

image

**Review by Tomcowan1 — APPROVED** [review]

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me!

**Pull request merged**